cpp/src/converters/renumber.cuh

```diff
@@ -31,134 +31,111 @@
 #include <thrust/scan.h>
 #include <thrust/binary_search.h>
 #include <cudf/cudf.h>
+#include <nvstrings/NVStrings.h>
 #include <cuda_runtime_api.h>
 
 #include "utilities/error_utils.h"
 #include "utilities/graph_utils.cuh"
-#include "utilities/heap.cuh"
+#include "sort/bitonic.cuh"
 #include "rmm_utils.h"
 
+namespace thrust {
```

Comment:
Should this be in thrust's namespace? If this is just for cugraph, it may better be in cugraph::detail namespace (or anonymous namespace if used only within a single translation unit following cudf's convention). Someone seeing thrust::swap(pair0, pair1) may mistakenly think this is defined in thrust.

Reply:
I will check and address in my next push.  It strikes me it needs to be in the thrust namespace because it is overloading a templated function that is defined in the thrust namespace.  But I will see if I can put it somewhere else, my last exploration of swap was back a few iterations of the C++ standard.

Reply:
I can't get any other mechanism to compile, although if you know another way to override the thrust::swap method (the default method generates an incorrect answer) I'd be happy to change this.

Reply:
The issues has been marked as blocked and moved to release 0.10, so it should *NOT* be closed.